scripts/build-lynx-trace-output.sh

```diff
@@ -0,0 +1,56 @@
+# Copyright 2024 The Lynx Authors. All rights reserved.
+# Licensed under the Apache License Version 2.0 that can be found in the
+# LICENSE file in the root directory of this source tree.
+
+#!/bin/bash
```

Comment:
shebang should be the first line of the script. And using `#!/usr/bin/env bash` is a better choice.